packages/cashier/src/app.jsx

```diff
@@ -13,7 +14,9 @@ const App = ({ passthrough: { WS, root_store } }) => {
 
     return (
         <MobxContentProvider store={root_store}>
-            <Routes />
+            <StoreProvider store={root_store}>
+                <Routes />
+            </StoreProvider>
         </MobxContentProvider>
```

Comment:
Right now `MobxContentProvider` and `StoreProvider` are almost the same things, But as we move forward with MobX 6 we will be removing `MobxContentProvider` as it's not needed anymore.

Reply:
what do you mean by we can remove MobxContentProvider?
what you have done is to just use the hook methodology to implement this `StoreProvider`?

Reply:
@ali-hosseini-fs `MobxContentProvider` is coming from `packages/cashier/src/stores/connect.js` which contains legacy code related to the older version of MobX, Moving to MobX 6 we won’t need this `connect` function in our components to access the props anymore and we can just wrap them with `observer` from MobX.

So instead of maintaining the legacy file, I decided to add a new one with a single responsibility, And then we can remove `MobxContentProvider` safely in the future.